bench/ibench/workloads/iibench_driver.py

```diff
@@ -0,0 +1,99 @@
+import os
+import sys
+import math
+
+from iibench import apply_options, run_main, run_benchmark
+
+def run_with_params(params):
+    # Extracting parameters from the dictionary
+    apply_options_only = params['apply_options_only']
```

Comment:
I'm not sure what the value of initializing with a params object is here... This funciton is also intended to be called in a standalone way.

Reply:
The value is in adding more parameters if needed in a cleaner way that doesn't require positional arguments. Also, it helps with the case the we have multiple calls to `run_with_params` with one change in the list of params (similar to the one in `benchmark`). In this case, we can make it more clear what's the parameter we're changing.